src/pages/dar_application/TypeOfResearch.js

```diff
@@ -170,19 +170,19 @@ export const TypeOfResearch = hh(class TypeOfResearch extends Component {
             disabled: props.disabled,
           }),
 
-          props.other ?
-            textarea({
-              style: otherTextStyle,
-              value: props.otherText,
-              onChange: props.otherTextHandler,
-              name: 'otherText',
-              id: 'otherText',
-              maxLength: '512',
-              rows: '2',
-              required: props.other,
-              placeholder: 'Please specify if selected (max. 512 characters)',
-              enabled: true
-            }) : undefined
+          textarea({
+            isRendered: props.other,
+            style: otherTextStyle,
+            value: props.otherText,
+            onChange: props.otherTextHandler,
+            name: 'otherText',
+            id: 'otherText',
+            maxLength: '512',
+            rows: '2',
+            required: props.other,
+            placeholder: 'Please specify if selected (max. 512 characters)',
+            enabled: true
```

Comment:
One last comment, and I apologize for not catching it earlier. This template is a small part of an application form that we show before and after submission. Before submission it makes sense to edit (its a form after all), but afterwards the information should be read-only, so we need to make the textarea is disabled.

HTML inputs generally have a `disabled` property that determines if an input should be editable or not. On top of that this file has the value `props.disabled` which carries the needed boolean value. So simply swap out `enabled: true` with `disabled: props.disabled` and you should be good to go.

One more note: The styling may not look like a disabled element when you do this and that's expected. Textareas require some custom styling on our part to achieve that look, but that's outside the scope of this PR so don't worry about tackling that here!

Reply:
that makes sense! I will fix that